bin/index.js

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const commandLineUsage = require('command-line-usage')
+const commandLineArgs = require('command-line-args')
+const directoryTree = require('../lib/directory-tree');
+
+const optionList = [
+    {
+        name: 'path',
+        alias: 'p',
+        required: true,
+        defaultOption: true,
```

Comment:
@mihneadb i totally agree with you about default argument! Fortunately, it easy  🙂

Reply:
Thanks!